dev-scripts/deploy_custom_rh-che.sh

```diff
@@ -39,14 +39,12 @@ export RH_CHE_RUNNING_STANDALONE_SCRIPT="false";
 function setVars() {
   if [ "$RH_CHE_IS_V_FIVE" == "true" ]; then
```

Comment:
do we need to support Che 5 ? I would prefer to remove it completely from scripts

Reply:
@l0rd ^

Reply:
We don't need to support Che 5 anymore

Reply:
Che 5 support officially dropped.